elastic/kibana
# [Uptime] Convert monitor sparklines to histogram

## Summary

We are changing the visualization of check counts from line charts to stacked histograms to make a more consistent feel for the UI and because it looks much better.

![image](https://user-images.githubusercontent.com/18429259/51126508-5610a780-17f1-11e9-85ba-7816b3411c30.png)


## Testing this PR
- Ensure you have some heartbeat data in your index
- Load the _Overview_ page and make sure you see stacked bars instead of lines

### Checklist

Use ~~strikethroughs~~ to remove checklist items you don't feel are applicable to this PR.

~~- [ ] This was checked for cross-browser compatibility, [including a check against IE11](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#cross-browser-compatibility)~~
~~- [ ] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/README.md)~~
~~- [ ] [Documentation](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#writing-documentation) was added for features that require explanation or tutorials~~

### For maintainers
